v22.2/cluster-setup-troubleshooting.md

```diff
@@ -304,6 +304,34 @@ The solution is to add connection retry logic to your application.
 
 ## Clock sync issues
 
+#### A node's timezone data has changed
+
+CockroachDB relies on each node's operating system timezone database to resolve named timezones like `America/New_York`, and for applying daylight saving rules. This can lead to inconsistent results across a cluster when coercing date/time strings into temporal types.
+
+Ensure that each node's operating system uses the same timezone database. In general, Cockroach Labs recommends that each node has an identical hardware and software configuration.
+
+Timezone data is read from the following sources in order of preference:
+
+1. The `ZONEINFO` operating system environment variable, if it is set to point to a `zoneinfo.zip` file such as [the one included in Go](https://github.com/golang/go/tree/master/lib/time) or to the directory that results from unzipping a `zoneinfo.zip`.
```

Comment:
This phrasing makes the reader think that it's Go who decides the format. That's not the right sentiment.

The format is standardized - it's called "tzdata" and it's a file hierarchy where all systems (except windows) do the same. The Go runtime supports loading this data *either from a directory* which is the most common way or from a zip file -- with the zip simply containing a copy of the tzdata in the standard format.

Here are the most common scenarios, **from the end user's perspective** and in decreasing order of commonality:

1. the tzdata is in one of the standard locations (/usr/share/zoneinfo etc). In this case they have nothing to do.
2. the tzdata is provided by the OS or the container base image, in a non-standard directory. In this case they set ZONEINFO to point to that directory and call it a day.
3. they are building a custom container (via their own Dockerfile or whatnot) and *they didn't know they needed a copy of tzdata*. In this case, we need to remind them that they should include "tzdata installation" as one of the build steps for their container image, usually via the base image package manager. Otherwise things won't work.
4. the user is working on a size-restricted environment (e.g. small containers or whatnot) and they want a "smaller tzdata database" that's just sufficient for the needs of their apps. In this case, they can take a copy of the tzdata on a standard system; trim it down to just what they need; put the result in a  zip file, then make the crdb deployment use that via the ZONEINFO env var.

Of course use case (1) and (2) are far more common than (3). And case (4) is just exotic.

Finally, there's the windows case. We just needs to tell them that:
- on windows, go tries to emulate tzdata on a "best effort" basis using the tz information in the windows registry. There may be discrepancies with unix systems. 
- if they really want the same tzdata as on unix, the user can take a copy of the tzdata directory on a unix system, put that copy on windows, and point to it via the ZONEINFO env var. They can either copy the full directory with all the files, or just a subset, or perhaps bundle all this in a zip file. We don't have an opinion.